app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -224,8 +225,8 @@ def verify_credentials(args)
       )
 
       metrics_username, metrics_password = metrics_authentication&.values_at("userid", "password")
-      metrics_port, metrics_database = metrics_endpoint&.values_at(
-        "metrics_port", "metrics_database"
```

Comment:
After the DDF form changes the `args["endpoints"]["metrics"]` hash looks like `{"hostname"=>"localhost", "port"=>5432, "path"=>"ovirt_engine_history"}` we no longer prefix with the endpoint name because we aren't mashing all keys into one flat hash anymore.